### PR TITLE
Updated new graph images

### PR DESCRIPTION
**Pull request by ghost**
- Updated Train Map Graph image
- Updated Social Graph image

### CODE REVIEW

File: static/img/graphs/train_map.png
Comment (dustinlarimer) [inline static/img/graphs/train_map.png:1]: What do the edges represent here? Distance between stations, or transit time? Can we add some light annotation so the image doesn't need to rely as much on the surrounding page copy?

**Review by diana-macrometa — APPROVED** [review]

**Comment by diana-macrometa** [comment]
@dlozina-macrometa or @matt-macrometa, can you address Dustin's question?

File: static/img/graphs/train_map.png
Comment (ghost) [reply static/img/graphs/train_map.png:1]: Edges are actually travel time. I agree that it is not clear at the first glance. We will correct the image. Thanks! cc @matt-macrometa

**Comment by diana-macrometa** [comment]
What's the status on this, guys? Will it be merged soon?

**Comment by diana-macrometa** [comment]
@dustinlarimer and @dlozina-macrometa, does anything else need to happen with this, or can we merge it?

**Comment by ghost** [comment]
Yes, we can. Good to go, @diana-macrometa. Sorry for the delay

**Pull request merged**